solution/ui/regulations/eregs-component-lib/src/components/JumpTo.vue

```diff
@@ -0,0 +1,158 @@
+                    title="Regulation section number, i.e. 111"
+                    aria-label="Regulation section number, i.e. 111"
+                />
+            </div>
+            <input
+                id="jump_btn"
+                class="submit"
+                :class="{ active: isActive }"
+                type="submit"
+                value="Go"
+                @click="getLink"
+            />
+        </div>
+    </div>
+</template>
+<script>
+import { getTitles, getParts } from "../api";
+
+export default {
+    name: "JumpTo",
+
+    props: {
+        apiurl: {
+            type: String,
+            required: true,
+        },
+        title: {
+            type: String,
+            required: false,
+            default: "",
+        },
+        part: {
+            type: String,
+            required: false,
+            default: "",
+        },
+    },
+    data() {
+        return {
+            titles: [],
+            selectedPart: "",
+            defaultTitle: "",
+            selectedTitle: "",
+            selectedSection: "",
+            hideTitle: false,
+            filteredParts: [],
+            isActive: false,
+            parts: [],
+            link: "",
+            base:
+                import.meta.env.VITE_ENV && import.meta.env.VITE_ENV !== "prod"
+                    ? `/${import.meta.env.VITE_ENV}`
+                    : "",
+        };
+    },
+
+    async created() {
+        // When title 45 or another title is added uncomment line below, and remove the hardcoded
+        // this.titles = await getTitles(this.apiurl);
+        this.titles = ["42"];
+
+        if (this.titles.length === 1) {
+            this.selectedTitle = this.titles[0];
+            this.defaultTitle = this.selectedTitle;
+            this.hideTitle = true;
+        }
+        if (this.title !== "") {
+            this.selectedTitle = this.title;
+            this.hideTitle = true;
+        }
+        if (this.part !== "") {
+            this.selectedPart = this.part;
+            this.isActive = true;
+        }
+    },
+
+    watch: {
+        selectedTitle(title) {
+            if (title === "") {
+                this.selectedPart = "";
+                this.isActive = false;
+            } else {
+                this.getParts(title);
+            }
+        },
+
+        selectedPart(part) {
+            if (part !== "") {
+                this.isActive = true;
+            } else {
+                this.isActive = false;
+            }
+        },
+    },
+
+    methods: {
+        async getParts(title) {
+            const partsList = await getParts(this.apiurl, title);
+            this.filteredParts = partsList.map((part) => part.name);
+        },
+        getLink() {
+            let link = `${this.base}/goto/?title=${this.selectedTitle}&part=${this.selectedPart}`;
+            if (this.selectedSection !== "") {
+                link += `&section=${this.selectedSection}&${this.selectedpart}-version='latest'`;
```

Comment:
`selectedpart` needs to be camelCased here; it's showing up as `undefined` in the `goto` URL.  Which is actually still taking you to the right place lol

Reply:
I actually didn't realize this `goto` url path existed -- I love that you can specify `latest` and it will go to the latest version.  I feel like I should have used this elsewhere in the Search/Resources pages.

Reply:
Im thinking it works because even though we have it say part-version="latest" we dont need it to.

Reply:
Actually this worked because it was undefined!  just want -version=latest, no part number